docs/js-core-2/week-1/homework.md

```diff
@@ -4,6 +4,190 @@ title: Coursework
 sidebar_label: Coursework
 ---
 
+## 0) Flipped Classroom Practice (3h)
+
+The video playlist contains some "content" videos and one "practice" video for each content video.
+
+0. Define an object. Complete the following code, assigning objects to the variables `salem` and `ijeoma` so that the output of the program is as below. Don't change the function!
+
+```js
+let salem = {
+  // fill in object here
+};
+
+// create an object here and assign it to a variable called ijeoma
+
+// This function is provided. You should not modify it!
+// It takes as input a person -> how it uses the fields
+// in the person object will tell you what fields to put in the
+// objects above
+function logInformationAbout(person) {
+  let age = 2023 - person.yearOfBirth;
+  console.log(
+    `${person.firstName} is from ${person.countryOfOrigin}`,
+    `and will be ${age} years old in 2023`
+  );
+}
+
+logInformationAbout(salem);
+logInformationAbout(ijeoma);
+```
+
+The output should be
+
+```
+Salem is from Sudan and will be 30 years old in 2023
+Ijeoma is from England and will be 22 years old in 2023
+```
+
+1. 1.1 Predict, evaluate and explain the following expressions in the node REPL
+
+```js
+let stock = {
+  name: "aapl",
+  lastClosingPrice: 171.45,
+  lastFiveDaysClosingPrices: [180.33, 176.28, 175.64, 172.99, 171.45],
+};
+
+stock.name;
+stock["name"];
+stock.lastClosingPrice * 100;
+stock.lastFiveDaysClosingPrices[1];
+
+stock.name = "AAPL";
+
+stock;
+```
+
+2. Add a field (a key + value pair) to the stock object, describing it's change over the last 5 days
+
+```js
+let stock = {
+  name: "aapl",
+  lastClosingPrice: 172.99,
+  lastFiveDaysClosingPrices: [179.99, 180.33, 176.28, 175.64, 172.99],
+};
+
+// write code here to add the changeInLastFiveDays field to stock
+
+console.log(
+  "In the last 5 days, the stock changed by",
+  stock.changeInLastFiveDays
+);
+```
+
+The output should be
+
+```
+In the last 5 days, the stock changed by -7
+```
+
+3.1 Write a program that prints out to the console information from the following object
```

Comment:
3.1 Write a program that logs a string to the console. Get the information from the following object: